read_it/tests/setup_data.py

```diff
@@ -0,0 +1,10 @@
+from ..models import Book
+
+
+def create_new_book(id: int | None = None) -> Book:
+    return Book.objects.create(
+        id=id,
+        title=f"This is a test book number {id}",
+        description="This is a test description number",
```

Comment:
This is a test description or This is a test description of a book with and id number {id}. I think we do not provide here description number